php/request-replayer/Dockerfile

```
@@ -0,0 +1,18 @@
+FROM php:7.3
+
+RUN apt-get update \
+  && apt-get install -y git \
+  && apt-get install -y unzip \
```

This could be combined into one line if you see fit :)

```suggestion
  && apt-get install -y git unzip \
```